### ISSUE TITLE
[Spell/Interaction] Feign death + Readiness

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#3147]
https://www.wowhead.com/spell=5384/feign-death

There is a glitch with using these abilities together.

1. Readiness is usable while you feign and it does not break feign.
**How it should work:** any action you do cancels feign death, readiness should be no exception.

2. Using Readiness while still feigning causes feign's cd to bug out. (related to 1)
Feign only starts ticking its cooldown when the effect is cancelled.
Readiness can reset the cooldown of feign even while it is not yet ticking (this is intended)
However since readiness does not break the effect, this means you are still feigned at this point, with feign's cd reset.
You can then recast feign without breaking it, staying feigned.
However, if you break the feign effect after reseting it with readiness, it goes on a hidden 30sec cooldown . (since the cd was reset, the visual display shows no cooldown, but you just can't use the ability for 30sec, it will say"not ready yet")
Basically the cd of feign resets and then somehow re-acquires the 30sec cd when you cancel the effect. You cannot track this cd due to the visual bug, which is confusing as hell.

**How it should work:** using readiness while feigning should break the feign effect and reset its cooldown

**Comment by Keader** [comment]
fixed

**Comment by Dranah** [comment]
Using readiness while feigning death now cancels the effect as intended and also properly resets its cooldown.
